src/component/VideoCard.js

```diff
@@ -1,8 +1,13 @@
-import React from "react";
+import { useState } from "react";
```

Comment:
you can write useState import along with React import

Reply:
yup